.editorconfig

```diff
@@ -5,3 +5,6 @@ charset = utf-8
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[*.test.js]
+trim_trailing_whitespace = false
```

Comment:
Oh no, was this breaking a test?

Reply:
Yeah, anywhere we want to compare against a string value like: `foo: ` that (intentionally) has trailing whitespace, it was being auto stripped. 😄

Reply:
Derp, sorry about that!